addons/resource.language.en_gb/resources/strings.po

```diff
@@ -6208,7 +6208,7 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#13435"
-msgid "Enable HQ Scalers for scalings above"
+msgid "Enable HQ scalers for scaling above"
```

Comment:
think the change is correct.
It shows as "Enable HQ scalers for scaling above 10%"  in settings.